code_samples/api/public_php_api/src/Command/TaxonomyCommand.php

```diff
@@ -41,7 +41,7 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $allEntries = $this->taxonomyService->loadAllEntries('tags', 30, 0);
 
-        $entry = $this->taxonomyService->loadEntryByIdentifier('desks');
+        $entry = $this->taxonomyService->loadEntryByIdentifier('desks', 'tags');
```

Comment:
Hi,

This is not mandatory. If not given, [`TaxonomyService::loadEntryByIdentifier`](https://github.com/ibexa/taxonomy/blob/main/src/lib/Service/TaxonomyService.php#L77) asks [`TaxonomyConfiguration::getDefaultTaxonomyName`](https://github.com/ibexa/taxonomy/blob/main/src/lib/Service/TaxonomyConfiguration.php#L86-L100) which returns [`'tags'`](https://github.com/ibexa/taxonomy/blob/main/src/lib/Service/TaxonomyConfiguration.php#L26). I confirmed that on a fresh install of a 4.5.3. If you have a bug making `'tags'` not being this default, can you, please, try to explain a bit more why you had to add this?

Reply:
For the example, we won't pass the default value of the default installation. But, as seen in your taxonomy_api.md change proposal, we'll mention that a second value can be passed when using another taxonomy.

Reply:
I am guessing here a bit, but I checked and in 4.0.8 that there, the parameter was not present and I have to admit that I only recently learned that identifiers were not globally unique (anymore) but only unique per taxonomy. Without fully setting up an older Ibexa DXP installation, I am sure that when testing taxonomy the first time last year, that was the case so that previously, there was no need for this parameter. I assume to keep backward compability, the parameter value was added as an optional value and "tags" was used as the best possible fallback parameter. But from a conceptual point, in my opinion, it does not really make sense to not expect the taxonomy name if the identifier is not globally unique. Then, a pair of taxonomy entry identifier and taxonomy name must always be given.

In multi-taxonomy setups, this code example of not specifying the taxonomy name might lead to sub-optimal code if in different taxonomies, the same identifier might be used.

I just think that the documentation code should contain as little obstacles as possible for code that will work in as many cases as possible.

It appears that only in `vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.4.latest-to-4.5.0.sql` (i.e. in the update from 4.4 to 4.5) was the constaint for global uniqueness of the taxonomy entry identifier column was removed. May-be this topic of what combination of taxonomy entry data (just identifier or identifier+taxonomy name) was unique in which version is something that is missing in the documentation and should be added if it is really missing in the documentation/release notes and not just something I missed?